wordpress-mobile/WordPress-iOS
# Feature/7675 social login analytics

Refs #7675 
Adds social login analytics.  These are the same as Calypso except for the following:
- "login_social_2fa_needed"
- "login_social_accounts_need_connecting"
- "login_social_error_unknown_user"

To test:
Test social login.
Ensure that each analytic is being bumped as intended.

Needs review: @nheagy
